mne/io/kit/kit.py

```diff
@@ -197,18 +199,18 @@ def _set_stimchannels(self, info, stim, stim_code):
 
             # modify info
             nchan = self._raw_extras[0]['nchan'] + 1
-            ch_name = 'STI 014'
-            chan_info = {}
-            chan_info['cal'] = KIT.CALIB_FACTOR
-            chan_info['logno'] = nchan
-            chan_info['scanno'] = nchan
-            chan_info['range'] = 1.0
-            chan_info['unit'] = FIFF.FIFF_UNIT_NONE
-            chan_info['unit_mul'] = 0
-            chan_info['ch_name'] = ch_name
-            chan_info['coil_type'] = FIFF.FIFFV_COIL_NONE
-            chan_info['loc'] = np.zeros(12)
-            chan_info['kind'] = FIFF.FIFFV_STIM_CH
+            chan_info = {
+                'cal':  KIT.CALIB_FACTOR,
+                'logno':  nchan,
+                'scanno':  nchan,
+                'range':  1.0,
+                'unit':  FIFF.FIFF_UNIT_NONE,
+                'unit_mul':  0,
+                'ch_name':  'STI 014',
+                'coil_type':  FIFF.FIFFV_COIL_NONE,
+                'loc':  np.zeros(12),
+                'kind':  FIFF.FIFFV_STIM_CH,
```

Comment:
These could be much more compact with `dict(cal=...)` style, might make browsing the file easier